src/Metal/MTLDevice.cs

```diff
@@ -42,6 +42,28 @@ public static partial class MTLDevice {
 				return system_default;
 			}
 		}
+
+#if MONOMAC
+		//[Mac (10,13, onlyOn64: true), NoiOS, NoWatch, NoTV]
+		//[DllImport (Constants.MetalLibrary)]
+		//static extern IMTLDevice[] MTLCopyAllDevicesWithObserver ([NullAllowed] out NSObject observer, MTLDeviceNotificationHandler handler);
```

Comment:
This results in a build failure for XM unified 32-bit (even though it's marked as 64 bit only).

```/Users/Tim/Projects/Xamarin/xamarin-macios-xcode9/xamarin-macios/src/Metal/MTLDevice.cs(49,98): error CS0246: The type or namespace name `MTLDeviceNotificationHandler' could not be found. Are you missing an assembly reference?```

The type in question is defined in metal.cs:

```delegate void MTLDeviceNotificationHandler (IMTLDevice device, MTLDeviceNotificationName notifyName);```

I'm not sure why it's giving the build error or how to fix it if anyone knows what's going on.